device/core/src/blob_upload/blob_upload_result.ts

```diff
@@ -52,7 +52,7 @@ export class BlobUploadResult {
     let uploadResult: BlobUploadResult;
     if (err) {
       const statusCode = ((Object.prototype.hasOwnProperty.call(err, 'statusCode')) ? (err.statusCode) : (-1));
-      const statusDescription = ((Object.prototype.hasOwnProperty.call(err, 'response')) ? (err.response) : (err.message));
+      const statusDescription = "failed to upload to blob";
```

Comment:
consider concatenating the StatusCode here.

Reply:
done @minune-msft